prac_05/state_names.py

```diff
@@ -9,10 +9,21 @@
 for code, name in CODE_TO_NAME.items():
     print(f"{code:3} is {name}")
 
-state_code = input("Enter short state: ").upper()
-while state_code != "":
-    if state_code in CODE_TO_NAME:
-        print(state_code, "is", CODE_TO_NAME[state_code])
-    else:
+# state_code = input("Enter short state: ").upper()
+# while state_code != "":
+#     if state_code in CODE_TO_NAME:
+#         print(state_code, "is", CODE_TO_NAME[state_code])
+#     else:
+#         print("Invalid short state")
+#     state_code = input("Enter short state: ").upper()
+
+is_valid_input = False
+while not is_valid_input:
```

Comment:
As per the prac requirements, does it say to repeat the inputs? I may be wrong about this though*

Reply:
I believe you are supposed to keep prompting the user until they enter an empty string, if that is what you are referring to.